src/test/java/uk/gov/justice/hmpps/casenotes/health/HealthCheckTest.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.justice.hmpps.casenotes.health;
+
+import org.junit.Test;
+import uk.gov.justice.hmpps.casenotes.controllers.ResourceTest;
+
+import static net.javacrumbs.jsonunit.assertj.JsonAssertions.assertThatJson;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class HealthCheckTest extends ResourceTest {
+    @Test
+    public void testDown() {
+        elite2MockServer.subPing(404);
+        oauthMockServer.subPing(404);
+
+        final var response = testRestTemplate.getForEntity("/health", String.class);
```

Comment:
Might be much cleaner serializing the response to a map and asserting against properties as you did above.

Reply:
not sure what you mean.  Above we serialise to a string and then use `JsonContent` to compare the contents of a file.  I'll merge this PR and then push a new version to see what you think